tests/test_2_login_with_correct_details.py

```diff
@@ -1,181 +1,49 @@
-    days_options.select_by_value(str(user_data["dob"][0]))
-
-    dob_months_dropdown = driver.find_element(By.XPATH, f'//select[@data-qa="months"]')
-    months_options = Select(dob_months_dropdown)
-    months_options.select_by_value(str(user_data["dob"][1]))
-
-    dob_years_dropdown = driver.find_element(By.XPATH, f'//select[@data-qa="years"]')
-    years_options = Select(dob_years_dropdown)
-    years_options.select_by_value(str(user_data["dob"][2]))
-
-    # Sign up for newsletter and receive special offers
-    newsletter_checkbox = driver.find_element(By.XPATH, '//input[@name="newsletter"]')
-    newsletter_checkbox.click()
-
-    special_offers_checkbox = driver.find_element(By.XPATH, '//input[@name="optin"]')
-    special_offers_checkbox.click()
-
-    # Fill out personal information
-    first_name_textbox = driver.find_element(By.XPATH, '//input[@data-qa="first_name"]')
-    first_name_textbox.send_keys(user_data["first_name"])
-
-    last_name_textbox = driver.find_element(By.XPATH, '//input[@data-qa="last_name"]')
-    last_name_textbox.send_keys(user_data["last_name"])
-
-    address_textbox = driver.find_element(By.XPATH, '//input[@data-qa="address"]')
-    address_textbox.send_keys(user_data["address"])
-
-    country_dropdown = driver.find_element(By.XPATH, '//select[@data-qa="country"]')
-    country_options = Select(country_dropdown)
-    country_options.select_by_value(user_data["country"])
-
-    state_textbox = driver.find_element(By.XPATH, '//input[@data-qa="state"]')
-    state_textbox.send_keys(user_data["state"])
-
-    city_textbox = driver.find_element(By.XPATH, '//input[@data-qa="city"]')
-    city_textbox.send_keys(user_data["city"])
-
-    zipcode_textbox = driver.find_element(By.XPATH, '//input[@data-qa="zipcode"]')
-    zipcode_textbox.send_keys(user_data["postcode"])
-
-    mobile_number_textbox = driver.find_element(
-        By.XPATH, '//input[@data-qa="mobile_number"]'
-    )
-    mobile_number_textbox.send_keys(user_data["phone"])
-
-    register_button = driver.find_element(
-        By.XPATH, '//button[@data-qa="create-account"]'
-    )
-    register_button.click()
-
-    # Verify that registration form is complete
-    account_created_text_present = element_present(
-        driver,
-        By.XPATH,
-        '//h2[@data-qa="account-created"]/b[contains(text(), "Account Created!")]',
-    )
-    assert account_created_text_present == True
-
-    # find and press continue button
-    continue_button = driver.find_element(By.XPATH, '//a[@data-qa="continue-button"]')
-    continue_button.click()
-
-    # find and press logout button
-    logout_button = driver.find_element(By.XPATH, '//a[@href="/logout"]')
-    logout_button.click()
+from ae_website.AEHomePage import HomePage
+from ae_website.AELoginPage import LoginPage
+from ae_website.AEAccountAction import AccountActionPage
+from ae_website.AESignUp import SignUpPage
 
 
 def test_case(init_driver, user_data):
```

Comment:
change test_case to test_happy_path

Reply:
added in c5c3282dbf65298092dd81c7212e9c231bf5978d